packages/dashboards/tests/acceptance/dashboards-test.js

```diff
@@ -246,15 +244,15 @@ module('Acceptance | Dashboards', function(hooks) {
     await click('.add-widget .btn');
 
     assert
-      .dom('.add-widget-modal .btn')
+      .dom('.modal .button')
       .hasAttribute(
         'href',
         `/dashboards/1/widgets/new`,
         'Create new assigns the new widget route to the primary button'
       );
 
     await selectChoose('.report-select', 'Report 12');
-    await click('.add-widget-modal .btn');
+    await click('. .button');
```

Comment:
?
```suggestion
    await click('.modal .button');
```

Reply:
Sorry this is resolved in local I've been fixing the dashboard tests and I pushed to fix the lint when some stuff was a lil messed up